src/main/Bug/BugController.java

```diff
@@ -35,8 +36,16 @@ public BugController(MongoDatabase db) {
         logger.info("Trying to get fields from form");
         JSONObject req = new JSONObject(ctx.body());
         JSONObject res = new JSONObject();
-        String title = req.getString("bugTitle");
-        String description = req.getString("bugDescription");
+        String title = req.getString("title");
```

Comment:
You should create a service class called SubmitIssueService (you can also rename bug to issue everywhere) which implements Service